### PR TITLE
20 search by alcohol content web

### PR DESCRIPTION
**Pull request by marek-wilk**
* Changed SearchByAlcoholContent method so it uses now LINQ's queries
* Added SearchByAlcocolContent View which uses checkboxes to save user choices about Alcohol Content in Drinks
* Removed drinks_source.json from Console App (I did it while changing its destination, when Wojtek PR with database changes had another file directory - simply, I can add this file again to Console App if needed)

### CODE REVIEW

File: DrinkManagerConsole/drinks_source.json [hunk 1]
```diff
@@ -1,2314 +0,0 @@
-{
```
Comment (joanna-bukowska) [inline DrinkManagerConsole/drinks_source.json:1]: Why did you delete this file?

File: DrinkManagerWeb/Views/Drinks/SearchByAlcoholContent.cshtml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                </div>
+                <div class="form-check">
+                    @Html.CheckBoxFor(item => item.NonAlcoholics, new { @class = "form-check-input", @id = "nonAlcoholicsCheck" })
+                    <label class="form-check-label" for="nonAlcoholicsCheck">Non Alcoholic Drinks</label>
+                </div>
+                <div class="form-check">
+                    @Html.CheckBoxFor(item => item.OptionalAlcoholics, new { @class = "form-check-input", @id = "optionalAlcoholicsCheck" })
+                    <label class="form-check-label" for="optionalAlcoholicsCheck">Optional Alcohol Drinks</label>
+                </div>
+                <button>Search</button>
```
Comment (joanna-bukowska) [inline DrinkManagerWeb/Views/Drinks/SearchByAlcoholContent.cshtml]: Could you use the same "Search" button and add "Clear" button as we have in search by ingredients and by name pages?

File: DrinkManagerWeb/Views/Drinks/SearchByAlcoholContent.cshtml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        <header class="d-flex align-items-center">
+            <h1>Search by Alcohol Content</h1>
+        </header>
+        @*Checkboxes returns user choice of alcohol content to controller. 
+            They are automatically checked when user comes to this view from another view*@
+        <div>
+            <form asp-route-pageNumber="1" method="post">
+                <div class="form-check">
+                    @Html.CheckBoxFor(item => item.Alcoholics, new { @class = "form-check-input", @id = "alcoholicsCheck" })
+                    <label class="form-check-label" for="alcoholicsCheck">Alcoholic Drinks</label>
```
Comment (joanna-bukowska) [inline DrinkManagerWeb/Views/Drinks/SearchByAlcoholContent.cshtml]: I think this: for="alcoholicsCheck" is redundant. There is info that "Cannot resolve id 'alcoholicsCheck'". I removed it here and in two other places down below and program works. But check it by yourself too.

File: DrinkManagerWeb/Views/Shared/_Layout.cshtml [hunk 4]
```diff
@@ -42,7 +42,7 @@
                     </li>
                     <li class="nav-item d-flex align-items-center mb-3 @(routeValues["Action"].ToString() == "Index" ? "active" : "")">
```
Comment (joanna-bukowska) [inline DrinkManagerWeb/Views/Shared/_Layout.cshtml]: In this line replace "Index" with "SearchByAlcoholContent".

**Review by joanna-bukowska — CHANGES_REQUESTED** [review]

File: DrinkManagerConsole/drinks_source.json (hunk 1 above)
Comment (marek-wilk) [reply DrinkManagerConsole/drinks_source.json:1]: I cut-pasted it to new directory, which was specified in Wojtek's PR when he was working on database. It is not necessary in DrinkManagerConsole, so I decided not to leave it there, but as I mentioned above - in my comment about my PR - I can add it if you would prefer it that way.

File: DrinkManagerWeb/Views/Drinks/SearchByAlcoholContent.cshtml (hunk 2 above)
Comment (marek-wilk) [reply DrinkManagerWeb/Views/Drinks/SearchByAlcoholContent.cshtml]: Done!

File: DrinkManagerWeb/Views/Drinks/SearchByAlcoholContent.cshtml (hunk 3 above)
Comment (marek-wilk) [reply DrinkManagerWeb/Views/Drinks/SearchByAlcoholContent.cshtml]: Still working, done!

File: DrinkManagerWeb/Views/Shared/_Layout.cshtml (hunk 4 above)
Comment (marek-wilk) [reply DrinkManagerWeb/Views/Shared/_Layout.cshtml]: Done!

**Review by marek-wilk — COMMENTED** [review]
I've implemented things which Joanna pointed out.

File: DrinkManagerConsole/drinks_source.json (hunk 1 above)
Comment (joanna-bukowska) [reply DrinkManagerConsole/drinks_source.json:1]: No, you dont't have to add it. I was just curious why this was deleted. :)

File: DrinkManagerWeb/Views/Drinks/SearchByAlcoholContent.cshtml (hunk 2 above)
Comment (joanna-bukowska) [reply DrinkManagerWeb/Views/Drinks/SearchByAlcoholContent.cshtml]: If you add some space between choice options, Search and Clear buttons and showing details, that will be great. :)

**Review by joanna-bukowska — APPROVED** [review]

**Review by wojciechkapko — APPROVED** [review]
